index.html

```diff
@@ -33,6 +33,293 @@
 </head>
 
 <body>
+  <section id="abstract">
```

Comment:
As a follow-up PR, I expect we'll want to specify a bit more:
1. What the policy could say.
2. What kind of warning the UA needs to show, and when. (This session might be recorded? This session is recorded? What if the app toggles recording briefly on then off at regular intervals? Should the warning to the user last at least N seconds or something?)

Reply:
added in the privacy indicator requirements setction.